tests/redisUnit.js

```diff
@@ -0,0 +1,199 @@
+function quitDB(client) {
+  return new Promise((resolve) => {
+    client.flushdb(() => {
+      // never fails
+      resolve();
+    });
+  })
+  .then(() => new Promise((resolve) => {
+    client.quit(() => {
+      // do not check for errors, just resolve
+      resolve();
+    });
+  }));
+}
+
+define((require) => {
+  const bdd = require('intern!bdd');
+  const assert = require('intern/chai!assert');
+  // This modifies the node module loader to work with es2015 modules.
+  // All subsequent `require` calls that use the node module loader
+  // will use this modified version and will be able to load es2015
+  // modules.
+  require('intern/dojo/node!babel-core/register');
+
+  const engine = require('intern/dojo/node!../../../../engine/index').test;
+  const redis = require('intern/dojo/node!../../../../engine/redis-helper').default;
+
+  // Create a sub-suite with `bdd.describe`. Sub-suites can
+  // have their own sub-suites; just use `bdd.describe`
+  // within a suite.
+  //
+  // Use `bdd.before` to define a function that will
+  // run before the suite starts, `bdd.after` to define a
+  // function that will run after the suite ends, `bdd.beforeEach`
+  // to define a function that will run before each test or sub-suite,
+  // and `bdd.afterEach` to define a function that will run after each
+  // test or sub-suite.
+  //
+  // Use `bdd.it` to define actual test cases.
+  //
+  // Within a test, throwing an `Error` object will cause the test to fail.
+  // Returning a promise will make the test async; if the promise
+  // eventually resolves then the test will pass. If the promise
+  // eventually rejects then the test will fail. Reject with a descriptive
+  // `Error` object please.
+  //
+  // Within a test, `this` refers to a test suite object. You can use it
+  // to skip the test or do other test-specific things.
+  //
+  // `this.remote` is null for unit tests.
```

Comment:
nit: This comment block no longer exists in our other test files. Please remove this and add a comment block at the top of this file based on the one in nodeUnit.js